core/webviz/_webviz.py

```diff
@@ -46,7 +47,9 @@ def __init__(self, title, icon=None):
 
     def add_content(self, content):
         """
-        Add a :class:`PageElement` to the page.
+        Add a :py:class:`PageElement` to the page.
+
+        :param content: The :py:class:`PageElement` to add.
```

Comment:
Should also document that it raises ValueError if content is not an instance of PageElement.